src/com/nisovin/magicspells/spells/targeted/PainSpell.java

```diff
@@ -96,7 +96,8 @@ public String getSpellDamageType() {
 	
 	private boolean causePain(Player player, LivingEntity target, float power) {
 		if (target.isDead()) return false;
-		double resolvedValue = damageExpression.resolveValue(null, player, player.getLocation(), target.getLocation()).doubleValue();
+		// TODO: This is a pretty crappy fix for http://prnt.sc/d9q6cv, but honestly idk
+		double resolvedValue = damageExpression.resolveValue(null, player, player.getLocation(), (target != null ? target : player).getLocation()).doubleValue();
```

Comment:
The we actually shouldn't even reach this point with a null target. The more likely case is for the caster to be null.